qa/tasks/cephfs/test_quota.py

```diff
@@ -104,3 +104,50 @@ def test_remote_update_write(self):
         with self.assertRaises(CommandFailedError):
             self.mount_b.write_n_mb("subdir_data/file", 40)
 
+    def test_human_readable_quota_values(self):
+        self.mount_a.run_shell(["mkdir", "subdir"])
+
+        self.assertEqual(self.mount_a.getfattr("./subdir",
+                                               "ceph.quota.max_bytes"), None)
+
+        # kilobytes
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "10k")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "10240")
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "100K")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "102400")
+
+        # megabytes
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "10m")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "10485760")
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "100M")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "104857600")
+
+        # gigabytes
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "2g")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "2147483648")
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "4G")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "4294967296")
+
+        # terabytes
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "1t")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "1099511627776")
+        self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "2T")
+        self.assertEqual(self.mount_a.getfattr(
+            "./subdir", "ceph.quota.max_bytes"), "2199023255552")
+
+        # invalid arguments
+        with self.assertRaises(CommandFailedError):
+            self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "10A")
+        with self.assertRaises(CommandFailedError):
+            self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "1y00K")
+        with self.assertRaises(CommandFailedError):
+            self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "G")
+        with self.assertRaises(CommandFailedError):
+            self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "")
```

Comment:
One more input to test is `af00`, alphanumeric but not ending with a alphabet.

Reply:
added, thanks!